### ISSUE TITLE
Simple intro to the command line for Windows users

### ISSUE THREAD

**Issue by fragglet** [chocolate-doom/chocolate-doom#810]
We should probably provide a wiki page that gives a brief introduction to the command line for Windows users who have never had to use it before. See [this thread](https://www.doomworld.com/vb/doom-general/91614-fast-monsters-fast-not-working-for-me/) for an example of why. There might also be a good existing guide that we can link to, although I suspect there's advantage in having a guide tailored specifically to Chocolate Doom.

It only needs to cover really basic things: drives and directory paths, the `cd` command and how to run Chocolate Doom. Abundance of screenshots would also be a good idea. Should not cover non-Windows platforms.

Alternative is that we should do #599 and start shipping a launcher on Windows. That's probably preferable in the long term.

**Comment by chungy** [comment]
Windows 10 is complicating things as the latest beta builds are removing the cmd.exe shell and replacing it entirely with PowerShell. This won't cause issues on the most basic commands, but some things will differ -- for example, the current working directory is *not* in the %PATH% in PowerShell, unlike cmd.exe. Some escaping and redirection features are also fundamentally different.

At least until that change makes it to a regular release, we can probably deal without worrying about it. @CapnClever has expressed interest in writing a basic Windows launcher using nothing but the Win32 API. I hope that the plan actually becomes a reality :-)

**Comment by fx02** [comment]
AFAIK cmd still exists and works, it's just not default.

**Comment by jmtd** [comment]
I thought powershell relied upon cmd.exe -- cmd.exe provides a "terminal" (for lack of a better word) within which runs a DOS-like shell, or Powershell.

**Comment by chungy** [comment]
No, it's a common misconception. The actual console window code resides in user32.dll and cmd.exe is merely just one program that can use it. PowerShell is wholly independent of it.

**Comment by jmtd** [comment]
> No, it's a common misconception. The actual console window code resides in user32.dll and cmd.exe is merely just one program that can use it. PowerShell is wholly independent of it.

I see, thanks.

The depressing thing is that means the console window code will live on into the future, even though it's horrid and has been for decades.
